tbx/core/blocks.py

```diff
@@ -2,18 +2,6 @@
 from django.utils.functional import cached_property
 
 from wagtail import blocks
-from wagtail.blocks import (
-    BooleanBlock,
-    CharBlock,
-    FieldBlock,
-    PageChooserBlock,
-    RawHTMLBlock,
-    RichTextBlock,
-    StreamBlock,
-    StructBlock,
-    StructValue,
-    URLBlock,
-)
```

Comment:
**Question**: I didn't follow the reasoning for this change. The commit says it's to "harmonize usage". Is this so that all the imported blocks are namespaced, and distinguishable from locally-declared blocks?

Reply:
Yes, that was the idea. The commit message should have been clearer